src/adservice/Dockerfile

```diff
@@ -1,9 +1,17 @@
-FROM openjdk:8
-RUN GRPC_HEALTH_PROBE_VERSION=v0.1.0-alpha.1 && \
-    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
-    chmod +x /bin/grpc_health_probe
+FROM openjdk:8-alpine as base
+
+FROM base as builder
+
 WORKDIR /app
 
+# This is because the protoc-gen-grpc-java library 
+# is a glibc compiled binary and alpine is
+# based on musl-libc
+RUN  apk add --no-cache --allow-untrusted \
+    -X https://apkproxy.herokuapp.com/sgerrand/alpine-pkg-glibc \
```

Comment:
why do we need the untrusted package?

Reply:
I think we pull the .pub/.apk as shown in the linked repo and avoid --allow-untrusted? That would be a good improvement over current one.

Reply:
👍